modules/node_modules/@ncigdc/components/SurvivalPlotWrapper.js

```diff
@@ -31,26 +32,31 @@ const styles = {
   },
 };
 
-function onMouseEnterDonor(e, donor) {
-  const tooltip = document.querySelector('.global-tooltip');
-  tooltip.classList.add('active');
-  tooltip.innerHTML = `
-    ${donor.id}<br>
-    Survival Rate: ${Math.round(donor.survivalEstimate * 100)}%<br>
-    ${donor.censored ? `Interval of last follow-up: ${donor.time}` : `Time of Death: ${donor.time}`}
-  `;
+function onMouseEnterDonor(e, donor, dispatch) {
```

Comment:
How is this(and onMouseLeaveDonor) getting dispatch, I don't see anything hooking it up?